rllib/core/learner/learner.py

```diff
@@ -233,13 +237,20 @@ def __init__(
         self._module_spec = module_spec
         self._module_obj = module
         self._optimizer_config = optimizer_config
-        self._hps = learner_hyperparameters
+        self._hps = learner_hyperparameters or LearnerHyperparameters()
 
         # pick the configs that we need for the learner from scaling config
-        self._distributed = learner_scaling_config.num_workers > 1
-        self._use_gpu = learner_scaling_config.num_gpus_per_worker > 0
+        self._learner_group_scaling_config = (
+            learner_group_scaling_config or LearnerGroupScalingConfig()
+        )
+        self._distributed = self._learner_group_scaling_config.num_workers > 1
+        self._use_gpu = self._learner_group_scaling_config.num_gpus_per_worker > 0
         # if we are using gpu but we are not distributed, use this gpu for training
-        self._local_gpu_idx = learner_scaling_config.local_gpu_idx
+        self._local_gpu_idx = self._learner_group_scaling_config.local_gpu_idx
+
+        self._framework_hyperparameters = (
```

Comment:
This property was missing entirely! The incoming `framework_hyperparameters` arg was ignored.